ui-frontend/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter.structure.tsx

```diff
@@ -46,9 +47,27 @@ export const serialize = {
     if (unit === undefined || !parseFloat(last)) {
       return ''
     }
-    const prefix = unit === 'm' || unit === 'h' ? 'PT' : 'P'
+    //Weeks is not a valid unit, so convert this to days
+    if (unit === 'w') {
+      let convertedUnit = 'd'
+      let convertedLast = (parseInt(last) * 7).toString()
```

Comment:
✏️  I feel like it might not be safe to assume this, so I might want to check if there's a moment function that does this for us.